ait/core/server/broker.py

```diff
@@ -81,6 +87,52 @@ def _subscribe_all(self):
                 else:
                     self._subscribe(subscriber, plugin.name)
 
+        ## Lastly setup the outputstream to receive commands
+        self._subscribe_cmdr()
+
+    def _subscribe_cmdr(self):
+        """
+        Setup for the appropriate outbound stream that is configured to
+        accept command messages.  If no stream is specified, it looks
+        for the first outbound stream.
+        """
+
+        ## Only continue if command topic was supplied
+        if not self.command_topic:
+            return
```

Comment:
Let's dump an error message here. This seems like a bad state to end up in quietly

Reply:
On second thought, do we even care about this? It's not really possible that we end up triggering this block. If something really odd happened and `ait.DEFAULT_CMD_TOPIC` wasn't available I think the failure when we try to subscribe is acceptable. Thoughts?

Reply:
I prefer to keep _some_ level of sanity checking in place - even when the design says they shouldn't be necessary.  So I went ahead and added the error log message.

Reply:
Works for me!